src/ResourceManager/Sql/Commands.Sql/help/Get-AzureRmSqlDatabaseBackupLongTermRetentionPolicy.md

```diff
@@ -25,13 +25,41 @@ The policy is an Azure Backup resource used to define backup storage policy.
 
 ## EXAMPLES
 
-### 1:
+### Example 1: Get the current version of the long term retention policy
+```powershell
+PS C:\> Get-AzureRmSqlDatabaseBackupLongTermRetentionPolicy -ResourceGroupName resourcegroup01 -ServerName server01 -DatabaseName database01 -Current
+
+
+ResourceGroupName                      : resourcegroup01
+ServerName                             : server01
+DatabaseName                           : database01
+WeeklyRetention                        : P2W
+MonthlyRetention                       : PT0S
+YearlyRetention                        : PT0S
+WeekOfYear                             : 0
+State                                  :
+RecoveryServicesBackupPolicyResourceId :
+Location                               :
 ```
 
-```
+This command gets the current version of the long term retention policy for database01
 
 ## PARAMETERS
 
+### -Current
+Whether or not to use the legacy Long Term Retention Vaults.
```

Comment:
This comment sounds sortof backwards

Reply:
Another case of the help file not generating. I had to delete the old one and regenerate for it to work...